iroh/src/client/blobs.rs

```diff
@@ -55,6 +61,38 @@ impl<C> Client<C>
 where
     C: ServiceConnection<RpcService>,
 {
+    /// Check if a blob is completely stored on the node.
+    ///
+    /// Note that this will return false for blobs that are partially stored on
+    /// the node.
+    pub async fn status(&self, hash: Hash) -> Result<BlobStatus> {
+        let status = self.rpc.rpc(BlobStatusRequest { hash }).await??;
+        Ok(status.0)
+    }
+
+    /// Check if a blob is completely stored on the node.
+    ///
+    /// This is just a convenience wrapper around `status` that returns a boolean.
+    pub async fn has(&self, hash: Hash) -> Result<bool> {
```

Comment:
not sure about this, but it sure is convenient...

Reply:
love it